qa/Gemfile

```
@@ -1,5 +1,5 @@
 source "https://rubygems.org"
-gem "rspec", "~> 3.1.0"
+gem "rspec", "~> 3.5"
```

Why `~> 3.5`?

An optimistic lock on 3.5 will currently resolve to latest 3.13.x, to any future 3.x while ensuring we _don't_ consume 4.x, so I don't see a reason to allow a future change to the dependency graph to roll this back as far as 3.5.x.

```suggestion
gem "rspec", "~> 3.13"
```

---

I was thinking to align on core rspec definition (make sure at the given runtime, they consume same rspec version): https://github.com/elastic/logstash/blob/main/Gemfile.template#L37
However, from current situation, I think `3.13` makes sense.